static/css/app.css

```diff
@@ -108,6 +108,26 @@ h1, h2, h3, h4, h5, h6 {
   width: 12.5rem;
 }
 
+#speech-button {
+  position: fixed;
```

Comment:
I just landed an .icon-button CSS class in app.css which will let you remove most of this CSS if you apply it to the speech-button element.

Reply:
Fixed.